src/uniparc/config/UniParcColumnConfiguration.tsx

```diff
@@ -182,13 +194,14 @@ UniParcColumnConfiguration.set(UniParcColumn.sequence, {
   ),
 });
 
+type XRefWithDatabase = UniParcXRef &
+  Required<Pick<UniParcXRef, 'database' | 'id'>>;
 UniParcColumnConfiguration.set(UniParcColumn.accession, {
   label: 'UniProtKB',
   render: ({ uniParcCrossReferences }) => (
     <ExpandableList descriptionString="entries" displayNumberOfHiddenItems>
       {uniParcCrossReferences
-        ?.filter((xref): xref is UniParcXRef &
-          Required<Pick<UniParcXRef, 'database' | 'id'>> =>
+        ?.filter((xref): xref is XRefWithDatabase =>
           Boolean(xref.database?.startsWith('UniProtKB') && xref.id)
```

Comment:
Q about this syntax, as I don't think we've been using it so far - would it be good practice to add this to guidelines too? especially when dealing with `null`, `undefined` in conjunction with other statement tests

Reply:
Which syntax specifically? TypeScript's ` is ` statement with `.filter()` in order to do some type guards?
We already use it in about 9 instances throughout the codebase, so even if not really common, it's not too new.
Not sure how to specify its use in the guidelines though, especially as I believe I'm not too used to using it and always doubt if I'm using it right, also, I'm always using it in conjunction with a `filter()`, so I'm not sure where else we can take advantage of it.

Reply:
No, I meant using  `Boolean()` instead of directly testing `xref.database?.startsWith('UniProtKB') && xref.id`

Reply:
Ah, it's just that xref.id is not a boolean, and TS wants the filter function to return a boolean (and not just truthy or falsy values)

Reply:
Ah ok, it's enforced by TS then.